packages/react/src/Label/Label.features.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import {ComponentMeta} from '@storybook/react'
+import {Label} from '.'
+import {CopilotIcon} from '@primer/octicons-react'
+
+export default {
+  title: 'Components/Label/Features',
+  component: Label
+} as ComponentMeta<typeof Label>
+
+export const Small = () => <Label size="small">Small</Label>
+
+export const Large = () => <Label size="large">Large</Label>
+
+export const Color = () => (
+  <>
+    <Label>Default</Label>
+    <Label color="blue">Blue</Label>
+    <Label color="blue-purple">Blue purple</Label>
+    <Label color="coral">Coral</Label>
+    <Label color="green">Green</Label>
+    <Label color="green-blue">Green blue</Label>
+    <Label color="gray">Gray</Label>
+    <Label color="indigo">Indigo</Label>
+    <Label color="lemon">Lemon</Label>
+    <Label color="lime">Lime</Label>
+    <Label color="orange">Orange</Label>
+    <Label color="pink">Pink</Label>
+    <Label color="pink-blue">Pink blue</Label>
+    <Label color="purple">Purple</Label>
+    <Label color="purple-red">Purple red</Label>
+    <Label color="red">Red</Label>
+    <Label color="red-orange">Red orange</Label>
+    <Label color="teal">Teal</Label>
+    <Label color="yellow">Yellow</Label>
+  </>
+)
+
+export const WithLeadingVisualSVG = () => (
+  <Label
+    leadingVisual={
```

Comment:
can we add a fill to the svg here? it's not visible in dark color mode
![Screenshot 2023-04-06 at 09 30 22](https://user-images.githubusercontent.com/13340707/230320912-8487cdfd-ab15-4024-b54e-5d80f2ffab71.png)

Reply:
Fixed 👍